trust-store-trust-policy-specification.md

```diff
@@ -158,41 +147,44 @@ The trust policy is represented as JSON data structure as shown below:
-- **`warn`**: This means implementation MUST perform the validation and if validation fails(because of any reason) the implementation MUST log an error and MUST NOT fail validation.
-- **`skip`**: This means implementation MUST NOT perform the validation.
-
-#### Scopes Constraints
+    For more information, see [registry scopes constraints](#registry-scopes-constraints) section.
+  - **`signatureVerification`**(*string*): This REQUIRED property dictates how signature verification is performed. Supported values are `strict`, `permissive`, `audit` and `skip`. Detailed explaination of each value is present [here](#signatureverification-details). A custom level can be defined by referencing a supported level value, and overriding individual validation checks.
+  - **`trustedIdentities`**(*object*): This REQUIRED property specifies a set of identities that the user trusts. For X.509 PKI, trusted identities is specified with a combination of trust store and trust anchors.
+    - **`trustStore`**(*string*): This REQUIRED property specifies a named trust store. Uses the format `{trust-store-type}:{named-store}`. Currently supported values for `trust-store-type` are `ca` and `tsa`. **NOTE**: When support for publicly trusted TSA is available, `tsa:publicly-trusted-tsa` is the default value, and implied without explictly specifying it. If a custom TSA is used the format `ca:acme-rockets,tsa:acme-tsa` is supported to specify it.
+    - **`identities`**(*array of strings*): This REQUIRED property specifies a list of elements/attributes of the signing certificate's subject. For more information, see [trusted identities constraints](#trusted-identities-constraints) section. A value `*` is supported if user trusts any identity (signing certificate) issued by the CA(s) in `trustStore`.
+
+#### Signature Verification details
+
+Notary v2 defines the following signature verification levels, to provide different levels of enforcement for different scenarios. Signature verrification is a multi step process that checks for the following - integrity (artifact is unaltered, signature is not currupted), authenticity (the signature is really from the identity that claims to have signed it), trusted timestamping (the signature was generated when the key/certificate were unexpired), expiry (an optional check if the artifact specifies an expiry date), revocation check (is the signing identity still trusted at present time). Based on the signature verification level, each of these checks is `enforced` or `logged`. `Enforced` causes signature verification to fail, and `logged` causes failures to be logged and the next check is evaluated. NOTE: Implementations may change the ordering of these checks, but all checks MUST to be performed for signature verification process to be considered complete.
```

Comment:
Two comments.  Normally expiry check is cheap and done first, and the MUST in the last sentence is odd.  Normally at the first failure you stop, but do you want to continue the rest of the checks to perform a full audit?  That though means potentially megacycles of CPU.

Reply:
That's good feedback, the last sentence isn't clear enough. Reworded and addressed typos.

```suggestion
Notary v2 defines the following signature verification levels, to provide different levels of enforcement for different scenarios. Signature verification is a multi step process performs the following validation checks - integrity (artifact is unaltered, signature is not corrupted), authenticity (the signature is really from the identity that claims to have signed it), trusted timestamping (the signature was generated when the key/certificate were unexpired), expiry (an optional check if the artifact specifies an expiry time), revocation check (is the signing identity still trusted at the present time). Based on the signature verification level, each of these validation checks is `enforced` or `logged`. If a validation check is `enforced`, a failure is treated as critical failure, and causes the overall signature verification to fail.  If a validation check is `logged`, a failure causes details to be logged, and the next validation check is evaluated till all checks succeed or a critical failure is encountered. NOTE: Implementations may change the ordering of these checks based on efficiency, but all validation checks MUST to be performed till the first critical failure is encountered, or all checks succeed, for signature verification process to be considered complete.
```